app/src/main/res/drawable/ic_quest_ferry_pedestrian.xml

```diff
@@ -0,0 +1,77 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
As per [this](https://wiki.openstreetmap.org/w/index.php?title=StreetComplete%2FQuests&type=revision&diff=1882295&oldid=1874863) we also need the SVG version of this icon. (`ic_quest_ferry_pedestrian`)

Respectively which color needs to be changed? To which color code? (AFAIK the background color.)

/cc @goldfndr

Reply:
Generally, the SVG version is outdated. Please always also change the SVG versions, not only the Android scalables.

Reply:
Okay, guessed and tinkled a little and I hope this is the correct icon: https://github.com/rugk/streetcomplete-quest-svgs/commit/91f65b7ef49b571b3c0620df4dd94d39c53dfd3d

Reply:
Yes, you are right. Thank you!